CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [0.14.5] - 2019-12-17
```

Comment:
Do not add this line. Once you merge the PR, the bot will automatically replace the `[Unreleased]` line by the actual version.